SConstruct

```diff
@@ -953,6 +953,10 @@ if env["minizip"]:
 if env["brotli"]:
     env.Append(CPPDEFINES=["BROTLI_ENABLED"])
 
+if env["d3d12"] and env["platform"] != "windows":
```

Comment:
That's an incorrect assumption, there are other platforms (not upstream) using Direct3D12.

What was the issue that you're trying to address? Because there's a lot more combinations if options which aren't valid for specific platforms than just this one (e.g. `vulkan` for web).

Reply:
If other platforms besides windows can use it, then that argument help should be updated to remove the "Windows only" note. Could be implemented as a supported platform flag, similar to `mono`

My use-case was the ability to swap a VSCode build task from windows to web by simply adding the `web=platform` option. Previously an error would occur in the d3d12 folder by an option not being available, not making explicit that the web platform simply didn't support the option